src/VisualStudio/Core/Def/Implementation/CommandBindings.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis.Editor.PlatformCommands;
+using Microsoft.VisualStudio.Editor.Commanding;
+using Microsoft.VisualStudio.LanguageServices;
+
+namespace Microsoft.VisualStudio.Editor.Implementation
+{
+    internal sealed class CommandBindings
+    {
+        [Export]
+        [CommandBinding(Guids.RoslynGroupIdString, ID.RoslynCommands.GoToImplementation, typeof(GoToImplementationCommandArgs))]
+        internal CommandBindingDefinition gotoImplementationCommandBinding;
```

Comment:
I forget the layering here -- should this be somewhere lower in EditorFeatures? Or CommandBindingDefinition is the VS-only bit?

Reply:
Yes, binding is host specific so it's VS only thing, VS for Mac would need another way to bind.